benchmarks/device_uvector/device_uvector_bench.cu

```diff
@@ -15,7 +15,6 @@
  */
 
 #include "../synchronization/synchronization.hpp"
-#include "thrust/detail/raw_pointer_cast.h"
```

Comment:
oops, this must have been ICYM trying to be helpful. Anytime a non-local include is added in quotes, I blame automation. :)